pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -17,26 +17,21 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"log"
-
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apiserver/pkg/endpoints/request"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-
-	"sigs.k8s.io/cluster-api/pkg/apis/cluster"
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/common"
 )
 
-const ClusterFinalizer string = "cluster.cluster.k8s.io"
+const ClusterFinalizer = "cluster.cluster.k8s.io"
```

Comment:
Question: should this be `cluster.cluster.k8s.io` or `clusters.cluster.k8s.io`? I've seen that CRDs are named plural, but should plural name be used here as well?

Reply:
@xmudrii Good question.  Since this wasn't something changed by this PR and this PR has a lot of changes, perhaps it can be opened in a separate thread or issue.